config/initializers/riiif.rb

```diff
@@ -5,24 +5,22 @@
   # but we just want the id for the FileSet it's attached to.
```

Comment:
if we're removing RIIIF from donut, can't this initializer be removed as well?

Reply:
Nope, that gets added with the `hyrax:riiif` generator